databakerUtils/v4Functions.py

```diff
@@ -7,11 +7,6 @@
 import pandas as pd
 import numpy as np
 
-wholeDict = requests.get('https://api.beta.ons.gov.uk/v1/code-lists/administrative-geography/editions/one-off/codes').json()
-adminDict = {}
-for item in wholeDict['items']:
-    adminDict.update({item['code']:item['label']})
-del wholeDict
```

Comment:
you should also delete the function that uses this dict "AdminGeogLabelsFromCodes"